packages/peregrine/lib/talons/CartPage/GiftCards/useGiftCards.js

```diff
@@ -174,7 +174,9 @@ export const useGiftCards = props => {
 
     const submitForm = useCallback(
         values => {
-            const giftCardCode = values['card'];
+            // This is safe to trim because the field has an isRequired validator.
+            // By the time we get here, we know the field has a string value.
+            const giftCardCode = values['card'].trim();
```

Comment:
I was going to suggest auto trim by submit or by [mask](https://joepuzzo.github.io/informed/?path=/story/formatting--mask). Either works!

Reply:
Ooooh, I didn't even know about `mask` - thanks! 

Simple enough though that I'm just going to leave this one.